rcl_yaml_param_parser/test/benchmark/benchmark_variant.cpp

```diff
@@ -238,12 +230,7 @@ BENCHMARK_F(PerformanceTest, array_string_copy_variant)(benchmark::State & st)
     st.SkipWithError(rcutils_get_error_string().str);
   }
   src_variant.string_array_value->size = kSize;
-  for (size_t i = 0; i < kSize; i++) {
-    src_variant.string_array_value->data[i] = rcutils_strdup("string", allocator);
-    if (src_variant.string_array_value->data[i] == NULL) {
-      st.SkipWithError(rcutils_get_error_string().str);
-    }
-  }
+
```

Comment:
Unlike the other 3 copy loops you're removing, which are just modifying the values, this is populating the element value with something non-NULL. I think that will affect the copy operation's runtime, so let's keep it.

Reply:
Restored: https://github.com/ros2/rcl/pull/810/commits/b77b9802e45b5e0371c32e46ccf58cecf5e3af30